lib/debug_logging/instance_notifier_modulizer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module DebugLogging
+  module InstanceNotifierModulizer
+    def self.to_mod(methods_to_notify: nil)
+      Module.new do
+        Array(methods_to_notify).each do |method_to_notify|
+          # method name must be a symbol
+          opts = method_to_notify.is_a?(Array) && method_to_notify.last.is_a?(Hash) && method_to_notify.pop
+          if method_to_notify.is_a?(Array)
+            method_to_notify = method_to_notify.first&.to_sym
+          else
+            method_to_notify.to_sym
+          end
+          event_name = "#{method_to_notify}.log"
+          define_method(method_to_notify) do |*args, &block|
+            ActiveSupport::Notifications.instrument(event_name, opts) do
+              super(*args, &block)
+            end
+          end
+
+          ActiveSupport::Notifications.subscribe(event_name) do |*args|
+            Configuration.new.log do
```

Comment:
This `Configuration.new` reference appears to be a hack-to-make-it-work.  It should be using a `config_proxy` which allows it to use the most local config set by the user.  App-level, class-level, instance-level, or method-level.
Ref: https://github.com/pboling/debug_logging/blob/master/lib/debug_logging/instance_logger_modulizer.rb#L11

Reply:
Yeah, I had to put in `Configuration.new` as a temporary hack.  I'm using `config_proxy` now but probably not in the most efficient way.  Still digging into this part 👍